donkeycar/templates/complete.py

```diff
@@ -585,6 +585,21 @@ def run(self, mode, recording):
         cfg.AUTO_CREATE_NEW_TUB else cfg.DATA_PATH
     tub_writer = TubWriter(tub_path, inputs=inputs, types=types, metadata=meta)
     V.add(tub_writer, inputs=inputs, outputs=["tub/num_records"], run_condition='recording')
+    # Telemetry (we add the same metrics added to TubHandler
+    if cfg.HAVE_TELEMETRY:
+        from donkeycar.parts.telemetry import Telemetry
+
+        # publish all simple types
```

Comment:
This following code should probably go into a static function on your class, to keep the template cleaner.

Reply:
fixed